src/main/kotlin/io/github/mojira/arisa/modules/commands/RemoveVersionCommand.kt

```diff
@@ -0,0 +1,19 @@
+package io.github.mojira.arisa.modules.commands
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import io.github.mojira.arisa.domain.Issue
+import io.github.mojira.arisa.modules.ModuleError
+import io.github.mojira.arisa.modules.ModuleResponse
+import io.github.mojira.arisa.modules.assertTrue
+
+class RemoveVersionCommand : Command {
+    override fun invoke(issue: Issue, vararg arguments: String): Either<ModuleError, ModuleResponse> = Either.fx {
+        assertTrue(arguments.size > 1).bind()
+        val version = arguments.asList().subList(1, arguments.size).joinToString(" ")
+        assertTrue(issue.affectedVersions.any { it.name == version }).bind()
+        assertTrue(issue.project.versions.any { it.name == version }).bind()

```

Comment:
This line is probably not needed because it only matters whether the issue has the affected version. Additionally I doubt that an issue can have a version which does not exist as version for that project.